### ISSUE TITLE
Add Chroma?

### ISSUE THREAD

**Issue by jeffchuber** [axilla-io/ax#49]
Hi there, would you be open to adding Chroma as a vector store option?

**Comment by benjreinhart** [comment]
Hey Jeff, we actually initially shipped with support for Chroma. However, the Chroma js package was causing warnings and, IIRC, an issue with a module not being found when consuming it from a next.js application. Let me try again and provide more concrete feedback if I run into those problems again.

**Comment by jeffchuber** [comment]
@benjreinhart ah! that has been fixed now https://github.com/chroma-core/chroma/pull/929 - i will cut a new release of the `npm` package later today.

**Comment by jeffchuber** [comment]
@benjreinhart `1.5.6` should address this- https://www.npmjs.com/package/chromadb

**Comment by nichochar** [comment]
@jeffchuber if you want to tackle this work yourself, it shouldn't take too much time.
Here's the commit where we removed chroma because of the warnings, essentially, we simply need to reintroduce this code: https://github.com/axilla-io/ax/pull/37

We would love to have more contributors to the repo, and happy to help you through it and drive it to release.

**Comment by benjreinhart** [comment]
@jeffchuber we're also seeing the issues reported in https://github.com/chroma-core/chroma/issues/953 after updating to 1.5.6 (see #51). We'll probably hold off until that is resolved
